### PR TITLE
Role prefix in direct deployment

### LINKED ISSUE #368: To Alter Policy/Role Naming
**Describe the enhancement:**
Option for Altering the role/policy naming will help organisations to deploy this lambda with their IAM governance policies. 
**Describe a specific use case for the enhancement or feature:**
Ex - CManaged-Policy/Role name using a prefix of CManaged.

### PR DESCRIPTION
**Pull request by aspacca**
Enhancement
## What does this PR do?
Address code review changes required at #386 

## Why is it important?
Some organisation requires role/policy naming compliance for their deployment: see #368 

## Checklist

- [x] My code follows the style guidelines of this project
~- [ ] I have commented my code, particularly in hard-to-understand areas~
- [x] I have made corresponding changes to the documentation
~- [ ] I have made corresponding change to the default configuration files~
~- [ ] I have added tests that prove my fix is effective or that my feature works~
- [x] I have added an entry in `CHANGELOG.md`

## Author's Checklist

- [ ]

## Related issues

-
Closes #368

### CODE REVIEW

**Comment by aspacca** [comment]
@kamal1004 @hriprsd , please sign the CLA :)

thanks

**Comment by kamal1004** [comment]
@aspacca Signed the CLA 😃 
Thanks

**Comment by hriprsd** [comment]
Signed CLA 😄 . Thanks!

**Comment by aspacca** [comment]
@kamal1004 @hriprsd did you use your github handle and email to sign the CLA? :)

it seems the check didn't pass

**Comment by hriprsd** [comment]
> @kamal1004 @hriprsd did you use your github handle and email to sign the CLA? :)
> 
> it seems the check didn't pass

Signed now. Thanks!

**Comment by aspacca** [comment]
sorry @kamal1004, I had to rewrite the commit history and then again adding you back to the commits authors (I didn't want to let you out of the history as it was with my initial fixup :))

but now CLA check does not pass: can you please try to sign again with the email you used in the commit?

thanks 🙏

**Review by girodav — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,11 @@
+### v1.9.0 - 2023/06/08
+##### Features
+* Allow the possibility to set a prefix for role and policy when deploying with the `publish_lambda.sh` scriot: [#399](https://github.com/elastic/elastic-serverless-forwarder/pull/399)
```
Comment (girodav) [inline CHANGELOG.md:3]:
```suggestion
* Allow the possibility to set a prefix for role and policy when deploying with the `publish_lambda.sh` script: [#399](https://github.com/elastic/elastic-serverless-forwarder/pull/399)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,11 @@
...
+
+* https://github.com/elastic/elastic-serverless-forwarder/pull/399
```
Comment (girodav) [inline CHANGELOG.md:5]:
```suggestion
```

**Review by hriprsd — APPROVED** [review]

**Comment by aspacca** [comment]
@girodav typos fixed, thanks!

**Comment by hriprsd** [comment]
Looks like there's issue installing packages during docker build. 
`E: Package 'netcat' has no installation candidate
E: Unable to locate package openjdk-11-jdk`

Do we need to update the dockerfile here?

**Comment by aspacca** [comment]
hello @hriprsd 

> Looks like there's issue installing packages during docker build.

could you give me more context?
what os/platform are you running on, python version and command you run

thanks :)

**Comment by hriprsd** [comment]
> hello @hriprsd
> 
> > Looks like there's issue installing packages during docker build.
> 
> ~could you give me more context?~ ~what os/platform are you running on, python version and command you run~
> 
> found the error in the lint action, going to fix it
> 
> thanks :)

Any updates on this please? Thanks in advance!

**Comment by kamal1004** [comment]
Hy @aspacca 

Do we have any update?, waiting for things to get done for implementation.
Thanks  :-)

**Comment by hriprsd** [comment]
@aspacca @girodav 
Any updates on this? Is there something we could do to speed up the process? There's failure in code coverage test step.

**Review by zmoog — APPROVED** [review]
LGTM!

@girodav thank you for taking care of this one! 🙇

**Comment by girodav** [comment]
> @aspacca @girodav Any updates on this? Is there something we could do to speed up the process? There's failure in code coverage test step.

Thank you for rasing this up again. Between PTO and other priorities, it slipped through my radar 😞 .

**Comment by kamal1004** [comment]
@aspacca @girodav Thanks for the quick response 😃

**Pull request merged**